django_tenants/staticfiles/finders.py

```diff
@@ -8,6 +9,39 @@
 from django_tenants import utils
 from django_tenants.staticfiles.storage import TenantStaticFilesStorage
 
+class TenantAppDirectoriesFinder(AppDirectoriesFinder):
+    """
+    This is a replacement for the AppDirectoriesFinder. The only difference between
+    this and the standard AppDirectoriesFinder implementation is that this one will
+    understand how tenant-aware file handling works and where to search for your
+    files.
+
+    In-detail explanation:
+
+    Let's say that we have a Tenant called "Foo" (foo.com).
+    When we try to access, let's say, foo.com/admin, the browser will issue a request
+    to (among others) foo.com/static/foo/admin/css/base.css , instead of the
+    usual request if we didn't use django-tenants (foo.com/static/admin/css/base.css).
+
+    See the "foo" right before "admin"?
+
+    In order for this to work as expected we must remove the "foo", as we have
+    already added the path "tenants/%s/static" to the paths that Django will
+    look.
+
+    If we didn't remove the "foo" from the path, Django would look in
+    /code/tenants/foo/static/foo/admin/css/base.css , which is wrong.
+
+    Note: Keep in mind that this code won't run at all when behind uWSGI/NGINX/whatever
+    web server you're using. Django won't even know where your static files are at!
+    """
```

Comment:
Yes. Luckily, Django only saves the first file that it finds to ``STATIC_ROOT`` when we run ``collectstatic_schemas``.

If everything goes as planned then our tenant-specific static files will be found first (since these finders are configured higher up in ``STATICFILES_FINDERS``), and the tenant-specific versions of the static files will also be the ones served up by the web server.